lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -4795,6 +4795,12 @@ CompilerType SwiftASTContext::GetErrorType() {
   return {};
 }
 
+CompilerType SwiftASTContext::GetObjCObjectType() {
+  // FIXME: ClangImporter::Implementation stores this type, but it's not
+  // exposed.
+  return GetCompilerType(ConstString("$sSo8NSObjectCD"));
```

Comment:
is this ABI? If so could we return the symbol in the superclass instead of having both subclasses return/construct it?